File: Vincent OS/Arrêt.vb
Public Class Arrêt

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        ProgressBar1.Increment(2)
        If ProgressBar1.Value = 100 Then
            Timer1.Stop()
            Démarrage.Close()
            Bureau.Close()
            Login.Close()
            Bureau2.Close()
            Me.Close()
        End If
    End Sub
End Class

File: src/Vincent OS/Arrêt.cs
using System;
using System.Windows.Forms;

namespace Vincent_OS
{
    public partial class Arrêt
    {
        public Arrêt()
        {
            InitializeComponent();
        }

        private void Arrêt_Load(object sender, EventArgs e)
        {
            var backmusic = new System.Media.SoundPlayer(My.Resources.Resources.shutdownSound);
            backmusic.Play();
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            ProgressBar1.Increment(2);
            if (ProgressBar1.Value == 100)
            {
                Timer1.Stop();
                Close();
                Application.Exit();
                GC.Collect();
                GC.WaitForPendingFinalizers();
            }
        }
    }
}